internal/persiststorage/persiststorage_configmap.go

```diff
@@ -249,3 +249,45 @@ func (s *KubeConfigMapStore) ProcessChanges(ctx context.Context, dataMap **map[s
 
 	return
 }
+
+func (s *KubeConfigMapStore) ProcessChangesWithFunction(ctx context.Context, function ProcessFunc) (err error) {
+	raw_opt := metav1.SingleObject(metav1.ObjectMeta{
+		Namespace: s.nameSpace,
+		Name:      s.name,
+	})
+	opt := clnt.ListOptions{}
+	opt.Raw = &raw_opt
+
+	watcher, err := s.client.Watch(ctx, &corev1.ConfigMapList{}, &opt)
+
+	go func() {
+		for {
```

Comment:
Do we need some tests for this logic?

Reply:
You mean test.go? The subscription server will write the configmap. Alarm server will update the new map in the memory. I did test manually with 2 server instances (1 subscription, 1 notification) by adding/deleting the subscriptions. In the test.go, can we support 2 handler instances and mimic the configmap change? Please let me know.